benchmarks/python/requirements.txt

```diff
@@ -0,0 +1 @@
+litgpt[all]
```

Comment:
Can we make this requirement local?
We have several benchmark files that can run okay without this module. So this can be a hassle when trying to run unrelated benchmarks.

One way could be to use `@pytest.mark.skipif` to check for the presence of this module in relevant benchmarks.

Reply:
Sounds fair. I guess I should have stick to what the other benchmarks are doing and rely on the module installed in the container. At least we do have litgpt in our CI containers. I'll remove this file.